UnitystationLauncher/Models/Api/CharacterTokenResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace UnitystationLauncher.Models.Api;
+
+public class CharacterTokenResponse : JsonObject
+{
+    public string token { get; set; }
```

Comment:
codacy issues

Reply:
Problem is the rest of the code will then complain about nullble Even though it's neveer gonnna happen

Reply:
You can bypass this issue by just providing a default value. This why it will stop complaining about it, and you'll avoid accidental null passing

Reply:
> can't change the naming rules

You can use the ``[JsonPropertyName("token")]`` attribute to let the system know the name from the backend while keeping proper naming convention on c# land